### DISCUSSION TITLE
Automatic Properties Filling

### DISCUSSION THREAD

**Discussion by johnngondi** [spatie/laravel-data#748] [Q&A]
When I created my first Data "FacilityBlock" it automatically added properties in the `FacilityBlockData` class, I am not sure whether these properties were inferred directly from migration or $fillable property of the `FacilityBlock` model.
I have tried to recreate that to no success, what happened and how can I make the package start prefilling properties for me again?

**Comment by rubenvanassche** [comment]
We have no such feature to fill properties 😄 Maybe some AI assistant started doing this?

**Reply by johnngondi** [reply]
This seems to be a more reasonable explanation. I bet it is that Co-Pilot
